datasets/at/pep/crawler.py

```diff
@@ -0,0 +1,68 @@
+from dateutil.parser import parse
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def parse_german_date(date_string):
+    return parse(date_string, dayfirst=True, locale='de')
+
+def crawl_item(url_info_page: str, context: Context):
+
+    info_page = context.fetch_html(url_info_page)
+
+    first_name = info_page.xpath("//span[@itemprop='http://schema.org/givenName']/text()")[0]
+    last_name = info_page.xpath("//span[@itemprop='http://schema.org/familyName']/text()")[0]
+
+    person = context.make("Person")
+    person.id = context.make_id(first_name, last_name)
+
+    person.add("firstName", first_name)
+    person.add("lastName", last_name)
+
+    person.add("website", url_info_page)
+
+    try:
+        birth_date_in_german = info_page.xpath("//span[@itemprop='birthDate']/text()")[0]
+        person.add("birthDate", parse_german_date(birth_date_in_german))
+    except:
+        pass
+
+    try:
+        person.add("email", info_page.xpath("//a[@itemprop='http://schema.org/email']/text()")[0])
+    except:
+        pass
+
+    try:
+        person.add("phone", info_page.xpath("//a[@itemprop='http://schema.org/telephone']/text()")[0]).replace(' ', '')
+    except:
+        pass
+
+    position_name = info_page.xpath('//*[@id="app"]/main/div/div[4]/div[1]/div/div/div[2]/p/text()')[0]
```

Comment:
I think a less brittle way of selecting this would be

1. select the parent of the element containing the name
2. select the descendant of that parent which is a p tag with the bold class

index-based selectors (e.g. `div[4]`) can easily fail in surprising ways and either not select anything, or select something you didn't mean to, which is worse.

-----

Just for comparison, what would also be better than the index-based xpath, but more complicated and I don't think is the right thing for this iteration of this source, is looping over `.functionsText` descendants of `#mandate` for the position names? The date seems to be in the previous sibling, and you can probably add a check whether it contains .aktiv or .inaktiv

The benefit of this approach isn't just more detail about the position, it seems to also tap a bit more into a semantic selection of the element and a bit more safety in pulling out and emitting the position name

Reply:
Thanks for the feedback, I was just copying the chrome dev tools xpath, but I will try to use more generic paths in the other crawlers. Let me know if it looks good now.